app/views/candidate_interface/course_choices/options_for_provider.html.erb

```diff
@@ -6,11 +6,17 @@
     <%= form_with model: Provider.new, url: candidate_interface_course_choices_provider_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
       <%= f.govuk_error_summary %>
 
-      <h1 class="govuk-heading-xl">
-        <%= t('page_titles.which_provider') %>
-      </h1>
+      <%= f.govuk_radio_buttons_fieldset :code, legend: { size: 'xl', text: t('page_titles.which_provider') } do %>
+        <div class="govuk-!-margin-top-6">
+          <% @providers.map do |provider| %>
+            <%= f.govuk_radio_button :code, provider.code, label: { text: "#{provider.name} (#{provider.code})", size: 'm' } %>
```

Comment:
All radio options should look the same (normal weight, standard size). I think removing the `size` attribute should fix this.